src/binary/ibc-relayer/commands/start.ts

```diff
@@ -171,6 +176,20 @@ export async function start(flags: Flags, logger: Logger) {
     flags.scanFromDest,
     process.env.RELAYER_SCAN_FROM_DEST
   );
+  const enableMetrics =
```

Comment:
Why don't you use `resolveOption` here like everywhere else?

Ah, there is no support for boolean, only string and integer?
If that is the case, leave it for this PR and make an issue to add boolean support, pointing to this line

Reply:
Exactly!

https://github.com/confio/ts-relayer/issues/174